themes/default/content/cloud-engineering-summit/_index.md

```diff
@@ -17,39 +17,39 @@ workshops:
       icon: puzzle
       icon_color: violet
       link: /resources/build-reusable-infrastructure-as-code-patterns
-      times: [ "2021-10-21T09:00:00-07:00" ]
+      times: [ "2021-10-21T12:00:00-07:00" ]
       description: |
         In this workshop, we’ll guide you through an example of building a reusable Pulumi component for a hypothetical “production ready application” in Python.
 
     - title: How to Build a Self-Service Cloud with Python
```

Comment:
I don't see this one on the spreadsheet unless it's the Automation API one? If it is, then this is the right time.

Reply:
"how to build a self-service cloud with python" is indeed the automation API one

Reply:
sessionize is probably more accurate than the spreadsheet for content :)

Reply:
yeah this is totally wrong; this is a new workshop from Lee. so the links, etc, are incorrect. it is "how to author and publish pulumi packages" but it's linking the reusable code patterns (that is a different workshop)

this one has never been given before so there won't be anything in /resources, etc

Reply:
We currently don't have a story for publishing Pulumi packages either, so we should circle back on this with @gthuang to make sure we have the right story here

Reply:
Until we get this one ironed out, I'm going to leave it in here as is as that's what's in the sheet (just to avoid changing something without changing our source of truth). I can always open a PR in a few minutes to adjust it.